docs/conf.py

```diff
@@ -147,6 +147,29 @@
 # so we do not mock tensorflow so we do not need to extend the logic there.
 autodoc_mock_imports = ['paddle', 'torch', 'torchvision']
 
+
+# -- Options for MathJax -----------------------------------------------------
+
+# Configure the MathJax to use SVG output as a default instead of the
```

Comment:
Instead of what?

Reply:
hmm, removed part of the comment by mistake when editing.
```suggestion
# Configure the MathJax to use SVG rendering as a default instead of the CHTML one.
```